modules/usage-of-latest-openshift-client.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * cicd/jenkins/important-changes-to-openshift-jenkins-images.adoc
+
+:_mod-docs-content-type: CONCEPT
+
+[id="about-the-openshift-cli-in-jenkins-images_{context}"]
+= About the OpenShift CLI tool in OpenShift Jenkins images
+
+[role="_abstract]
+If your Jenkins pipelines require a *specific* {oc-first} version for compatibility or reproducibility, you must explicitly configure the desired client version in the Jenkins pipeline DSL.
+
+Starting with 4.12, the OpenShift Jenkins container images ship with the *latest available version* of the `oc` CLI tool bundled inside the image, rather than the client version matching the cluster release.
+
+If your Jenkins pipelines require a *specific* `oc` client version for compatibility or reproducibility, you must explicitly configure the desired client version in the Jenkins pipeline DSL.
+
+// Module included in the following assemblies:
+//
+// * cicd/jenkins/important-changes-to-openshift-jenkins-images.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ocp-jenkins-release-oc-client-table_{context}"]
```

Comment:
you can use the id as the filename to keep things consistent; so make this a module that is called:

ocp-jenkins-release-oc-client-table.adoc

(same with the procedure; you can reuse the ID as the filename)